.github/workflows/release.yaml

```diff
@@ -0,0 +1,31 @@
+name: Release
+
+on:
+  create:
+    tags:
+      - 'v*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
+
+      - name: Get dependencies
+        run: make deps
+
+      - name: Build
+        run: make all
+
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        with:
+          files: |
+            tc-redirect-tap
```

Comment:
This will need to be multi-arch, to include arm64, otherwise this isn't something we are going to be able to use.

See:

https://github.com/alexellis/tc-tap-redirect-builder/releases/

https://github.com/alexellis/tc-tap-redirect-builder/tree/master/.github/workflows

Firecracker is built for ARM64 and x86_64

Reply:
You're right ! 

Can you have a look on this new version before I squash it ?